setup.cfg

```diff
@@ -0,0 +1,56 @@
+[metadata]
+# would we ever want to use pyproject.toml ?
+name = ccdhmodel
+# see home_page notes
+url = https://github.com/cancerDHC/ccdhmodel/
+# author metadata, license etc is totally incomplete
+# str
+# add maintainer
+author = Mark A. Miller
+author_email = MAM@lbl.gov
+# summary is alias for description ?
+summary = CRDC-H model in LinkML, developed by the Center for Cancer Data Harmonization (CCDH)
+description = CRDC-H model in LinkML, developed by the Center for Cancer Data Harmonization (CCDH)
+long_description = CRDC-H model in LinkML, developed by the Center for Cancer Data Harmonization (CCDH). This Python package contains the dataclasses necesarry to build and operate over data objects according to the CRDC-H model.
+# is this where mkdocs artifacts gets SENT?
+# or will this be used as a base for URLs in the mkdocs artifacts?
+# should change, but be careful not to overwrite the existing ccdhmodel docs yet
+# no, this is probably just what appears on the package's PyPI page
+# see also url
+# https://setuptools.readthedocs.io/en/latest/userguide/declarative_config.html says 
+#   home-page is an alias for url
+#   note hyphen vs underscore!
+home_page = https://turbomam.github.io/ccdhmodel_from_template
+license = CC0 1.0 Universal
+python_requires = >=3.7
+classifiers = 
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    Intended Audience :: Information Technology
```

Comment:
We should steal some classifiers from [biolink-model](https://pypi.org/project/biolink-model/), such as:
- Scientific/Engineering :: Bio-Informatics
- Intended Audience :: Healthcare Industry